self-tests.yml

```diff
@@ -721,6 +728,12 @@ tasks:
           permissions: public-read
           display_name: swagger.json (latest)
           patchable: false
+  - name: write-and-set-downstream-expansions
```

Comment:
I'd make sure this name includes that it's for pine. Something similar to what Minna said above makes sense!